src/ashley/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -1064,19 +1066,39 @@ msgstr ""
 msgid "Forum index"
 msgstr ""
 
-#: models.py:30
+#: editor/widgets.py:15
+msgid "Please enter your message here..."
+msgstr ""
+
+#: editor/widgets.py:16
+msgid "Enter or paste your URL here and press enter"
```

Comment:
Thanks ! I noted your suggestion and will address it in another PR, because it's not related to the rename feature.
It came here when I updated translations.